doc/rados/operations/placement-groups.rst

```diff
@@ -659,13 +659,17 @@ command of the following form:
 
    ceph osd pool set {pool-name} pg_num {pg_num}
 
-If you increase the number of PGs, your cluster will not rebalance until you
-increase the number of PGs for placement (``pgp_num``). The ``pgp_num``
-parameter specifies the number of PGs that are to be considered for placement
-by the CRUSH algorithm. Increasing ``pg_num`` splits the PGs in your cluster,
-but data will not be migrated to the newer PGs until ``pgp_num`` is increased.
-The ``pgp_num`` parameter should be equal to the ``pg_num`` parameter. To
-increase the number of PGs for placement, run a command of the following form:
+If you increase the number of PGs, your cluster will not be completely
+rebalanced it has increased the number of pools for placement (``pgp_num``).
+The ``pgp_num`` parameter specifies the number of PGs that the CRUSH algorithm
+considers for placement. 
+
+Increasing ``pg_num`` splits the PGs in your cluster, but data will not be
+migrated to the newer PGs until ``pgp_num`` is increased. 
+
+It is possible to manually set the ``pgp_num`` parameter. The ``pgp_num``
+parameter should be equal to the ``pg_num`` parameter. To increase the number
+of PGs for placement, run a command of the following form:
```

Comment:
We haven't had to touch `pgp_num` since Nautilus, the Manager I think is the component that steps it automagically when `pg_num` is changed.

Line 663 is grammatically ungood to boot, and the same dynamic applies if `pg_num` is reduced.

Suggest instead:

>Beginning with the Nautilus release Ceph automatically steps `pgp_num` for a pool whenever `pg_num` is changed, either by the PG autoscaler or manually.  Admins generally do not need to touch `pgp_num` directly, but can monitor progress with `watch ceph osd pool ls detail`.  When `pg_num` is changed, the value of `pgp_num` is stepped slowly so that the cost of splitting or merging PGs is amortized over time to minimize performance impact.

Reply:
I'm on it.

(new topic)
I raised this PR to respond to a complain on the ceph-users list. Let's leave this one open for a few days so that we can encourage discussion.

Reply:
Oh, Max!

Reply:
Missed it by that much.

Reply:
That's the third time this week!